### ISSUE TITLE
The animal model is really cool, I've incorporated it into my TensorRT Windows integration package

### ISSUE THREAD

**Issue by warmshao** [KwaiVGI/LivePortrait#301]
https://github.com/user-attachments/assets/385b86e8-1688-4838-9849-0ff965b1f343

You can directly get it from my project at: https://github.com/warmshao/FasterLivePortrait, and it also supports MediaPipe.

https://github.com/user-attachments/assets/dc37e2dd-551a-43b0-8929-fc5d5fe16ec5

**Comment by FurkanGozukara** [comment]
Amazing

I hope TensorRT can be merged into main repo

**Comment by tontheonelove** [comment]
@warmshao   bro ,  do you have guide to  complete install ,  I've tried many methods, but none of them worked.

i use windows 11

thankyou

**Comment by warmshao** [comment]
> @warmshao bro , do you have guide to complete install , I've tried many methods, but none of them worked.
> 
> i use windows 11
> 
> thankyou

You can use the Windows integration package I provided. It is portable, so you can simply unzip it and use it without installation.

**Comment by tontheonelove** [comment]
> > @warmshao bro , do you have guide to complete install , I've tried many methods, but none of them worked.
> > i use windows 11
> > thankyou
> 
> You can use the Windows integration package I provided. It is portable, so you can simply unzip it and use it without installation.

Bro , you mean  i unzip and just run 

For web usage: python app.py --mode trt --mp or python app.py --mode onnx --mp

No need install any ?

**Comment by warmshao** [comment]
> > > @warmshao bro , do you have guide to complete install , I've tried many methods, but none of them worked.
> > > i use windows 11
> > > thankyou
> > 
> > 
> > You can use the Windows integration package I provided. It is portable, so you can simply unzip it and use it without installation.
> 
> Bro , you mean i unzip and just run
> 
> For web usage: python app.py --mode trt --mp or python app.py --mode onnx --mp
> 
> No need install any ?

Yes, you don't need to install Python packages like TensorRT. However, you need to double-click a BAT file to convert ONNX to TRT. I've written detailed instructions in the README, please refer to that.

**Comment by tivikoff** [comment]
Hey, nice. @warmshao Is it possible in your implementation to make v2v with animal model? Because it looks like in the LivePortrait it is currently possible only for human models

**Comment by warmshao** [comment]
> Hey, nice. @warmshao Is it possible in your implementation to make v2v with animal model? Because it looks like in the LivePortrait it is currently possible only for human models

yes, it supports

https://github.com/user-attachments/assets/fef41fd0-8321-4357-869c-373ae6395d6a
